components/builder-depot/src/backend/s3.rs

```diff
@@ -210,6 +212,7 @@ impl S3Handler {
                 Counter::UploadFailures.increment();
                 debug!("{:?}", e);
                 let end_time = PreciseTime::now();
+                info!("Upload failed for: {}", &path_attr);
                 info!(
```

Comment:
I think we can remove the stat for an error case..

Reply:
Yeah so long as we have logging on a failure that's what matters most. I just wanted something to search on. :+1: